docs/src/language_servers.md

```diff
@@ -235,6 +235,28 @@ Follow installation instructions on [LSP-graphql](https://github.com/sublimelsp/
     }
     ```
 
+## Helm
+
+1. Install [helm-ls](https://github.com/mrjosh/helm-ls).
+2. (Optional & recommended) Install [yaml-language-server](https://github.com/redhat-developer/yaml-language-server).
```

Comment:
```suggestion
2. (Optional & recommended) Install integration with [yaml-language-server](https://github.com/mrjosh/helm-ls?tab=readme-ov-file#integration-with-yaml-language-server). 
```

Reply:
Your suggested wording doesn't make sense imo, but I replaced the link.

Reply:
Yes, it really doesn't make sense. Sorry about that.